README.md

```diff
@@ -54,6 +55,8 @@ Multi-schema consumers support multiple message types via handler configs. They
     * `options`, composed by
         * `handlers` – configuration for handling each of the supported message types. See "Multi-schema handler definition" for more details;
         * `messageTypeField` - which field in the message describes the type of a message. This field needs to be defined as `z.literal` in the schema and is used for routing the message to the correct handler;
+        * `messageTimestampField` - which field in the message contain the message creation date (by default it is `timestamp`). This field needs to a `Date` object or ISO-8601 date string;
+        * `maxRetryDuration` - how long (in seconds) the message should be retried due to the `retryLater` result before marking it as consumed (avoid infinite loops). Default is 4 days;
```

Comment:
```suggestion
        * `maxRetryDuration` - how long (in seconds) the message should be retried due to the `retryLater` result before marking it as consumed (and sending to DLQ, if one is configured). This is used to avoid infinite loops. Default is 4 days;
```